djallas/epicmail
# #164668143 Prepare App Version 2

## What does this PR do?
Prepare App Version 2

## Description
-Restructure directories to support 2 API versions

## How should this be manually tested?

Visit the following link to see swagger documentation based on API versions
http://epicmailbox.herokuapp.com/api/v1/docs
http://epicmailbox.herokuapp.com/api/v2/docs

## What are the relevant pivotal tracker stories?
###164668143
